spec/lib/tasks/fixtures/find-eu-exit-guidance-business.json

```diff
@@ -0,0 +1 @@
s as a result of EU Exit","document_type":"taxon","locale":"en","public_updated_at":"2018-09-07T07:55:41Z","schema_name":"taxon","title":"Brexit","withdrawn":false,"details":{"internal_name":"Brexit [P]","notes_for_editors":"","visible_to_departmental_editors":false},"phase":"live","links":{"parent_taxons":[{"api_path":"/api/content/government/all","base_path":"/government/all","content_id":"e48ab80a-de80-4e83-bf59-26316856a5f9","description":"","document_type":"taxon","locale":"en","public_updated_at":"2018-09-16T20:29:39Z","schema_name":"taxon","title":"Government","withdrawn":false,"details":{"internal_name":"Government","notes_for_editors":"","visible_to_departmental_editors":true},"phase":"live","links":{"root_taxon":[{"api_path":"/api/content/","base_path":"/","content_id":"f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a","description":"","document_type":"homepage","locale":"en","public_updated_at":"2019-06-21T11:52:37Z","schema_name":"homepage","title":"GOV.UK homepage","withdrawn":false,"links":{},"api_url":"https://www.gov.uk/api/content/","web_url":"https://www.gov.uk/"}]},"api_url":"https://www.gov.uk/api/content/government/all","web_url":"https://www.gov.uk/government/all"}]},"api_url":"https://www.gov.uk/api/content/government/brexit","web_url":"https://www.gov.uk/government/brexit"}],"available_translations":[{"title":"Find EU Exit guidance for your business","public_updated_at":"2019-06-19T11:22:02Z","document_type":"finder","schema_name":"finder","base_path":"/find-eu-exit-guidance-business","description":"This is the information that has been published so far for your business to prepare for EU exit. You can change what information you get using the checkboxes. Come back to this page regularly or sign up to receive emails when new information is published.","api_path":"/api/content/find-eu-exit-guidance-business","withdrawn":false,"content_id":"42ce66de-04f3-4192-bf31-8394538e0734","locale":"en","api_url":"https://www.gov.uk/api/content/find-eu-exit-guidance-business","web_url":"https://www.gov.uk/find-eu-exit-guidance-business","links":{}}]},"description":"This is the information that has been published so far for your business to prepare for EU exit. You can change what information you get using the checkboxes. Come back to this page regularly or sign up to receive emails when new information is published.","details":{"beta":false,"document_noun":"publication","summary":"This is the information that has been published so far for your business to prepare for EU exit. You can change what information you get using the checkboxes. Come back to this page regularly or sign up to receive emails when new information is published.","canonical_link":true,"sort":[{"name":"Topic","key":"topic","default":true},{"name":"Most viewed","key":"-popularity"},{"name":"Relevance","key":"-relevance"},{"name":"Most recent","key":"-public_timestamp"},{"name":"A to Z","key":"title"}],"filter":{"facet_groups":["52435175-82ed-4a04-adef-74c0199d0f46"]},"facets":[]}}
```

Comment:
Would it be possible to prettify this JSON, so it's not all on one line?

Also, I wonder whether the whole content item is required, or whether a small amount of test data (even with fake facet names) could be added?

Reply:
Hmmm I _could_ delete everything apart from `['links']['facet_group'][0]['links']['facets']` but would have to keep that hierarchy in order for the code to work on production. I envisage us removing this code once we've run the rake task once, so it didn't seem worth the effort to chop it around, plus this gives me confidence it will work against the production API and I won't have accidentally mangled it. I'd suggest leaving it un-prettified for the same reason.

Reply:
You can paste the json into https://jsonlint.com/ and it will prettify it for you.

I agree with cutting down the fixture to only what's needed. It will make the fixture, and therefore the tests easier to read.

Reply:
I'd also recommend using mock data (e.g. replacing actual content_id values with things like `a-facet-value`) in both the fixture and specs.  It then removes any misunderstanding in the future about whether the tests have to be updated when the rake tasks are changed to cater for different sector merges.